src/main/java/org/mdbenefits/app/data/enums/Counties.java

```diff
@@ -0,0 +1,38 @@
+package org.mdbenefits.app.data.enums;
+
+public enum Counties {
+  ALLEGANY("Allegany"),
```

Comment:
Minor - I would think (this is a question for Programs) that they would want the suffix County for all except Baltimore City:
1. Due to the fact that Baltimore City and Baltimore County are _both_ counties
2. Due to the fact that the counties with a possessive name sound weird without the suffix - e.g., Prince George's

Reply:
Yeah, I thought about this as well. The only annoying part is if we have to remove the word "County" when filling out forms.  Let's see what Programs thinks.

Reply:
I can add "County" in and then we can take it out, if need be.

Reply:
IIRC, this doesn't go on a form.